### PR TITLE
Fix form->add type=radio.

### PR DESCRIPTION
**Pull request by jeffutter**
Options wasn't being passed when adding a radio.. thus it wasn't making all the separate fields. Also there was no default tempalte and with no template it didn't render the labels. Still no way to set which field is 'radiod' by default.

### CODE REVIEW

File: classes/fieldset/field.php [hunk 1]
```diff
@@ -87,6 +87,7 @@ class Fieldset_Field
 	{
 		$this->name = (string) $name;
 		$this->fieldset = $fieldset;
+		if(isset($attributes['options'])) { $this->set_options($attributes['options']);}
```
Comment (jschreuder) [inline classes/fieldset/field.php:91]: This isn't up to our coding standards, no single line if statements.

File: classes/form.php [hunk 2]
```diff
@@ -637,13 +637,15 @@ class Form {
 			}
 		}
 
+		$options = $field->options();
+
```
Comment (jschreuder) [inline classes/form.php]: this isn't necessary for all, only used by select (directly) and radio/checkbox. As such this should be after the case statement.

File: classes/form.php [hunk 3]
```diff
@@ -725,7 +727,7 @@ class Form {
 					$bf_temp = str_replace('{label}', $label, $bf_temp);
 					$build_fields .= $bf_temp;
 				}
-				$template = str_replace($match[1], $build_fields, $template);
+				$template = str_replace($match[1], $build_fields, $match[1]);
```
Comment (jschreuder) [inline classes/form.php]: Shouldn't this be $template at the end? This way you're removing anything before and after the {fields} and {/fields}  tags.

**Comment by jeffutter** [comment]
Ok, i fixed the above issues. Hope that's good.

**Pull request merged**